test/bpf/verifier-test.sh

```diff
@@ -114,10 +113,6 @@ function cg_prog_type_init {
 	prog_types["_version"]="SKIP"
 	prog_types["version"]="SKIP"
 	if $BPFTOOL help 2>&1 | grep -q feature; then
-		if check_macro SOCK_OPS; then
```

Comment:
Thanks! From the comment, it looks like we can also remove the `prog_types[...]="SKIP"` at the beginning of this block.